compiler/main/driver.cpp

```diff
@@ -2309,7 +2309,7 @@ static chpl::CompilerGlobals dynoBuildCompilerGlobals() {
     .overloadSetsChecking = fOverloadSetsChecks,
     .divByZeroChecking = !fNoDivZeroChecks,
     .cacheRemote = fCacheRemote,
-    .privatization = !(fNoPrivatization || fLocal),
+    .privatization = !(fNoPrivatization || (fLocal && !usingGpuLocaleModel())),
```

Comment:
I didn't want to emphasize `--no-privatization` too much, because last time I tried to work with that flag I was getting errors. But that's a very old and probably stale information.